src/Service/Settings.cpp

```diff
@@ -206,7 +224,7 @@ void Settings::dump(WriteBufferFromOwnString & buf) const
     write_int(raft_settings->fresh_log_gap);
 }
 
-SettingsPtr Settings::loadFromConfig(const Poco::Util::AbstractConfiguration & config, bool standalone_keeper_)
+SettingsPtr Settings::loadFromConfig(const Poco::Util::AbstractConfiguration & config, bool standalone_keeper_, Poco::Logger * log)
```

Comment:
It is better to create a logger in `Settings .loadFromConfig` and name it to 'Settings'